### PR TITLE
updated docs to new api

### PR DESCRIPTION
**Pull request by ksaadDE**
Description of the issue/feature this PR addresses: new API route v2-swagger

Current behavior before PR: showing page does not exist in demo page

Desired behavior after PR is merged: points to new url

### CODE REVIEW

File: doc/rest_api.rst [hunk 1]
```diff
@@ -10,8 +10,9 @@ documentation. You will find them using the following url patterns:
 
 * API: *http://<hostname>/api/v1/*
 * Documentation: *http://<hostname>/docs/api/*
+* New Docs: *https://<hostname>/api/schema-v2/swagger/*
```
Comment (tonioo) [inline doc/rest_api.rst:13]: Could you please reference the old documentation as v1 and the new one as v2 ?

**Comment by ksaadDE** [comment]
@tonioo added the requested naming clarification (new, old API)

File: doc/rest_api.rst [hunk 2]
```diff
@@ -6,12 +6,18 @@ To ease the integration with external sources (software or other),
...
 Every installed instance comes with a ready-to-use API and a
-documentation. You will find them using the following url patterns:
+documentation. 
 
-* API: *http://<hostname>/api/v1/*
-* Documentation: *http://<hostname>/docs/api/*
+**You will find them using the following url patterns:**
 
-An `example <https://demo.modoboa.org/docs/api/>`_ of this
+* New API+Docs: *https://<hostname>/api/schema-v2/swagger/*
```
Comment (tonioo) [inline doc/rest_api.rst:13]: New API is also available under https://<hostname>/api/v2/

Comment (ksaadDE) [reply doc/rest_api.rst:13]: > New API is also available under `https://<hostname>/api/v2/`

At the time of editing this it wasn't available under that route.

**Pull request merged**